app/Controller/PagesController.php

```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```

Comment:
[staging] [code_sniffer] Each class must be in a namespace of at least one level (a top-level vendor name)